### ISSUE TITLE
Issues with Installing "Lethal Intelligence Experimental" Mod and Editing "Poltergeist" Configs

### ISSUE THREAD

**Issue by Foxy1873** [Kesomannen/gale#13]
Everything works well, and I really like this mod manager. However, I'm having an issue with installing a specific mod: "Lethal Intelligence Experimental" (https://thunderstore.io/c/lethal-company/p/VirusTLNR/LethalIntelligenceExperimental/).

The problem is that it simply won't download any files. The folder structure and a "plugin" file are created, but nothing else is downloaded. This also happens in the download cache, where only the "plugin" file is present.

To ensure it wasn't an issue with the mod itself, I tested it using the Thunderstore Mod Manager, and it downloaded without any problems there.

Additionally, I'm getting the following error after clearing the download cache and attempting to redownload it: Failed to execute 'install_mod' - Failed to install VirusTLNR-LethalIntelligenceExperimental: error while extracting mod (at C:\Users\jmach\AppData\Local\com.kesomannen.gale\cache\VirusTLNR-LethalIntelligenceExperimental\0.0.7): A file cannot be created if it already exists. (os error 183).

Another issue I noticed is when I try to edit the configs for the "Poltergeist" mod (https://thunderstore.io/c/lethal-company/p/coderCleric/Poltergeist/), everything becomes unclickable. I have to close and relaunch the program to be able to do anything again.

**Comment by Foxy1873** [comment]
"Lethal Intelligence Experimental" still doesn't seem to work with 0.5.1 but it seems to get a rewrite in a few days so this might solve it, Poltergeist config works fine now, thank you!

**Comment by Kesomannen** [comment]
I didn't do anything specific to solve it, but if it works it works 😅

**Comment by DropDaDeuce** [comment]
I am also experiencing issues installing "Lethal Intelligence Experimental". If I import a profile from R2 that already has it installed though, it seems to work fine.

**Comment by Kesomannen** [comment]
Oh wait nvm, ignore me

**Comment by Foxy1873** [comment]
Ok, thought the rewrite of the mod might fix it but nope, seems to be still an issue.
These are the files that are installed in two different modmanagers, Gale in this case not downloading or installing them at all i think.
```
Gale Mod Manager
└── VirusTLNR-LethalIntelligenceExperimental
    └── BepInEx
        └── plugins

R2Modman
└── VirusTLNR-LethalIntelligenceExperimental
    ├── CHANGELOG.md
    ├── icon.png
    ├── manifest.json
    ├── README.md
    └── VirusTLNR-LethalIntelligence
        ├── LethalIntelligence.dll
        └── mapdotanimpack
```

I tried a fresh install of Gale, removing every file i found from it from the computer and tried to install the Mod again, this time i got an error and it didn't install at all:

Failed to execute 'install_mod' - Failed to install VirusTLNR-LethalIntelligenceExperimental: error while extracting mod (at C:\Users\jmach\AppData\Local\com.kesomannen.gale\cache\VirusTLNR-LethalIntelligenceExperimental\0.0.8): A file cannot be created if it already exists. (os error 183).

After pressing download again i didn't get any error message and it was added to the profile with icon etc. but it still doesn't contain any files.


Something i realized, when i download it manually, and import it as a local mod in Gale everything seems to work fine.

**Comment by Kesomannen** [comment]
Currently looking into this, it seems to be an issue with the zip library I'm using. The "plugins" folder there is actually not actually a folder, but an empty file with no extension.

**Comment by Kesomannen** [comment]
Ok, I think I found the exact issue. The zip file is internally using a backslash for the `plugins` folder, which the zip library gets confused by and extracts it as a file instead.

**Comment by Kesomannen** [comment]
Fixed in 0.5.5
